### ISSUE TITLE
still compatible?

### ISSUE THREAD

**Issue by Datagrdn** [lukechampine/rote#10]
getting the following error when trying to run:

`syntax error at [61 5] in /app/rote/hoon
  /^  octs                 :: cast the following expression to the 'octs' type (not strictly necessary here, but good practice)
----^
[%error-building /app/rote/hoon]
: /~zod/home/26/app/rote/hoon`

wondering if its still compatible with current version of urbit

**Comment by lukechampine** [comment]
yeah this code is currently outdated, sorry. Updating it shouldn't be *too* bad; if someone else is willing to do the work and submit a PR, I'll send them a planet in appreciation. Otherwise it may be a bit before I get around to this.

**Comment by Datagrdn** [comment]
totally get it. im very new to hoon(hence trying to load the cards haha) but if you give me a diagnosis of what's wrong id be happy to dig in to fix it. would be a great exercise for me.

**Comment by lukechampine** [comment]
I like that attitude!

The error in this case stems from the fact that Ford was totally overhauled earlier this year. You will need to replace the old Ford runes (starting with `/`) with the [new ones](https://urbit.org/blog/ford-fusion/#ford-runes). Hopefully the correspondence is fairly clear.

btw, if you want to gauge your progress when learning Hoon, check out my [Evolution of a Hoon Programmer](https://gist.github.com/lukechampine/1b68178af5e05b30fb8e9efe96316514). :)

**Comment by Datagrdn** [comment]
Ahhh that's hilarious, thanks for sharing! I'll say that I... mostly get the joke, haha.

Was able to get the single file imports going with the new runes, however it looks like there's no Ford Fusion equivalent for /_ used to import multiple images. I posted in the urbit-dev group about it.

In the meantime I'll keep poking around, I'll need to do some more reading but could try importing each one since there are only two.

Psyched to get this going!
